projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -1603,11 +1603,15 @@
         }
 
         .sort-icon {
-            width: rem(15px);
-            height: rem(15px);
-            min-width: rem(15px); /* yeah IE, it really needs to be 15px wide... */
-            font-size: rem(15px);
             position: relative;
+            display: flex;
+
+            igx-icon {
+                width: rem(15px);
+                height: rem(15px);
+                min-width: rem(15px); /* yeah IE, it really needs to be 15px wide... */
```

Comment:
If this is IE specific, then it should be removed, since IE is not supported with 13. @simeonoff

Reply:
True, if deleting this line doesn't produce any artifacts in ever-green browsers, we can safely remove it.